jenkins-slaves/ansible/data/config/verdaccio/htpasswd

```diff
@@ -0,0 +1 @@
+root:2FKSJnJlTVx8s:autocreated 2020-12-21T08:27:52.149Z
```

Comment:
shouldn't we hide this in github secret?

Reply:
@mareknovotny moved to `htpasswd.j2` and password taken from `ENCRYPTED_PASSWORD` env variable (which is defined by the provisioning job)